packages/fx-core/src/plugins/resource/bot/utils/common.ts

```diff
@@ -240,6 +240,13 @@ export function convertToConstValues<V extends string, T extends { [key in strin
   return Object.values(targetValues).find((itemValue) => value === itemValue);
 }
 
+export function convertToEnumValues<T extends string>(
+  value: unknown,
```

Comment:
When we use Enum instead of constant string, maybe there is update for this method. The type of T and value, targetEnum looks a little weird. You can try below and check some related docs. https://www.technicalfeeder.com/2021/07/mastering-enum-in-typescript/

```
function getEnumByValue<T extends {[key:string]:string}>(enumType: T, enumValue:string): keyof T|undefined {
    return Object.keys(enumType).find(k => enumType[k] === enumValue);
}
```